### ISSUE TITLE
how to visualize the predicted densities of model 

### ISSUE THREAD

**Issue by aliraza0769** [weizheliu/Context-Aware-Crowd-Counting#2]
How to transform the multidimensional NumPy array into a 3-dimensional RGB image to visualize predicted densities of the model. The model outputs densities NumPy arrays with size: (1, 1, 45, 80), type: float32. Looking forward to your response. Thank you

**Comment by weizheliu** [comment]
Please refer to the [plot function](https://github.com/weizheliu/People-Flows/blob/main/plot.py) here for density map visualization.

**Comment by weizheliu** [comment]
I would assume this problem already solved, if you have more questions, please feel free to re-open it anytime.

**Comment by aliraza0769** [comment]
Thank you for providing us with technical assistance.  We tried to plot the density using `https://github.com/weizheliu/People-Flows/blob/main/plot.py` file but it give an error of importing CANNet2s. Since we trained the CANNet model defined in model.py file.  We are interested in plot density maps using the CANNet not using the CANNet2s. Then, is it still possible to generate density maps using CANNet model ?
